### PR TITLE
Comment cleanup in ERC721PoolCollateral.t.sol

### LINKED ISSUE #788: nit: incorrect comment
https://github.com/ajna-finance/contracts/blob/19cba1a432265ef6985e136b741af6e47be53fda/tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol#L89

`// borrower deposits one NFT into the pool`

### PR DESCRIPTION
**Pull request by ith-harvey**
## Description

* Cleaned up comments in `ERC721PoolCollateral.t.sol`

## Purpose
* Resolve https://github.com/ajna-finance/contracts/issues/788

## Tasks

- [ ] Changes to protocol contracts are covered by unit tests executed by CI.
- [ ] Protocol contract size limits have not been exceeded.
- [ ] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [x] Scope labels have been assigned as appropriate.
- [ ] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

File: tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol [hunk 1]
```diff
@@ -176,7 +163,7 @@ contract ERC721PoolCollateralTest is ERC721HelperContract {
             tokenIds: tokenIdsToAdd
         });
 
-        // check token balances after add
+        // check nFT balances after add
```
Comment (grandizzy) [inline tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol]: Nit NFT

File: tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol [hunk 2]
```diff
@@ -535,29 +507,29 @@ contract ERC721PoolCollateralTest is ERC721HelperContract {
...
-        // should revert if the actor does not have any LP to remove a token
+        // reverts when borrower2 does not have any LP to remove a NFT
         _assertRemoveCollateralInsufficientLPRevert({
             from:   _borrower2,
             amount: 1,
             index:  1530
         });
 
-        // should revert if we try to remove a token from a bucket with no collateral
+        // reverts when borrower trys to remove a NFT from a bucket with no collateral
```
Comment (grandizzy) [inline tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol]: Nit: tries

**Review by grandizzy — CHANGES_REQUESTED** [review]
2 nit to fix

File: tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol (hunk 2 above)
Comment (ith-harvey) [reply tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol]: fixed -> https://github.com/ajna-finance/contracts/pull/1044/commits/c91d048b83b56594be731d12bba38e0285fa812a

File: tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol (hunk 1 above)
Comment (ith-harvey) [reply tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol]: fixed -> https://github.com/ajna-finance/contracts/pull/1044/commits/c91d048b83b56594be731d12bba38e0285fa812a

**Review by grandizzy — APPROVED** [review]

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**